torch/_C/_distributed_c10d.pyi

```diff
@@ -408,3 +413,4 @@ def _verify_params_across_processes(
     params: List[Tensor],
     logger: Optional[Logger],
 ): ...
+def make_nccl_premul_sum(factor: Union[float, List[Tensor]]) -> ReduceOp: ...
```

Comment:
Can we underscore this function name for now? Because there is a chance this function be shuffled or name changed later.